rust-toolchain.toml

```diff
@@ -0,0 +1,3 @@
+[toolchain]
```

Comment:
I think we still have to add the wasm target to this toolchain as well because our clippy CI job depends on it.

Besides, we wanted to also have a CI job that checks the the WASM engine can still be compiled on stable (although clippy basically plays that role already).

Reply:
I think it still will be possible to use `rustup run`/`cargo _+stable` to force stable toolchain